lib/processors/versionInfoGenerator.js

```diff
@@ -23,9 +356,16 @@ function getTimestamp() {
  * @param {object} parameters.options Options
  * @param {string} parameters.options.rootProjectName Name of the root project
  * @param {string} parameters.options.rootProjectVersion Version of the root project
- * @param {Array} parameters.options.libraryInfos Array of objects representing libraries,
- *					e.g. <code>{name: "library.xy", version: "1.0.0"}</code>
- * @returns {Promise<module:@ui5/fs.Resource[]>} Promise resolving with an array containing the versioninfo resource
+ * @param {LibraryInfo[]} parameters.options.libraryInfos Array of objects representing libraries,
```

Comment:
The LibraryInfo type definition needs to be public as well to be visible in the JSDoc

Reply:
tackled with daf5610